package.json

```diff
@@ -27,7 +30,10 @@
   "devDependencies": {
```

Comment:
can you remove `querystring` as a dependency. it comes with node so it's useless to have it here.